src/main/java/org/apache/sling/feature/cpconverter/cli/ShutDownHook.java

```diff
@@ -76,4 +82,15 @@ public void run() {
         logger.info("+-----------------------------------------------------+");
     }
 
+    private void cleanUp(){
+        String tmpDir = System.getProperty("java.io.tmpdir");
+        logger.info("Cleaning up tmp directories {}, {}", tmpDir + "sub-content-packages", tmpDir + "syntethic-content-packages" );
```

Comment:
@kaushalmall - just a minor thing - tmpdir may not have separator at the end so the error message may actually not really return the right path, creating the the Dir fileobjects ahead and using the path out of those would be cleaner.

Reply:
@DominikSuess hopefully the latest change is better aligned to what you were thinking